test/Microsoft.AspNetCore.Hosting.FunctionalTests/ShutdownTests.cs

```diff
@@ -16,12 +15,7 @@ namespace Microsoft.AspNetCore.Hosting.FunctionalTests
 {
     public class ShutdownTests : LoggedTest
     {
-        private static readonly Regex NowListeningRegex = new Regex(@"^\s*Now listening on: (?<url>.*)$");
-        private const string ApplicationStartedMessage = "Application started. Press Ctrl+C to shut down.";
-
-        public ShutdownTests(ITestOutputHelper output) : base(output)
-        {
-        }
+        public ShutdownTests(ITestOutputHelper output) : base(output) { }
```

Comment:
Writing a unit test would be painful because we would need to attach to stdout, right?